fdbrpc/AsyncFileNonDurable.actor.h

```diff
@@ -190,11 +191,12 @@ class AsyncFileNonDurable final : public IAsyncFile, public ReferenceCounted<Asy
 	                    Reference<DiskParameters> diskParameters,
 	                    NetworkAddress openedAddress,
 	                    bool aio)
-	  : filename(filename), initialFilename(initialFilename), file(file), diskParameters(diskParameters),
-	    openedAddress(openedAddress), pendingModifications(uint64_t(-1)), approximateSize(0), reponses(false),
-	    aio(aio) {
+		: filename(filename), initialFilename(initialFilename), file(file), diskParameters(diskParameters),
+		  openedAddress(openedAddress), pendingModifications(uint64_t(-1)), approximateSize(0), reponses(false),
+		  aio(aio)
+		{
 
-		// This is only designed to work in simulation
+			// This is only designed to work in simulation
```

Comment:
This looks like a whitespace-only change that a proper clang format would not produce.